static/app/views/profiling/landing/profileCharts.tsx

```diff
@@ -132,7 +133,7 @@ export function ProfileCharts({query, router, selection}: ProfileChartsProps) {
                 axisLabel: {
                   color: theme.chartLabel,
                   formatter(value: number) {
-                    return axisLabelFormatter(value, 'count()');
+                    return axisLabelFormatter(value, aggregateOutputType('count()'));
```

Comment:
May as well hard code the actual type here instead of deriving it from a hard coded series name
```suggestion
                    return axisLabelFormatter(value, 'integer');
```